chat/src/main/java/com/twilio/video/app/chat/sdk/MessageWrapper.kt

```diff
@@ -0,0 +1,10 @@
+package com.twilio.video.app.chat.sdk
+
+import com.twilio.conversations.Message
+
+class MessageWrapper {
+
+    fun options(): Message.Options {
+        return Message.options()
+    }
```

Comment:
Do you anticipate having more functionality added to this wrapper class? Currently, it looks very light to require a class of its own.

Reply:
Yeah so these wrapper classes are an annoying necessity for mocking out code that depends on Android framework classes to avoid runtime exceptions in unit tests. Basically all of the classes under the `chat/sdk` directory are wrapper classes that allow the mocking. And yes, I plan to also add additional functions to mock out more functionality as we add it in future conversations work

Reply:
Got it, thanks for the details.